engine/src/exec-interface.cpp

```diff
@@ -3436,6 +3436,12 @@ void MCInterfaceExecPutIntoObject(MCExecContext& ctxt, MCExecValue p_value, int
         else
             t_text = (MCStringRef)p_chunk . mark . text;
         
+        if (MCStringIsEmpty(*t_text))
```

Comment:
I don't think this is a correct fix - I suspect it is t_chunk.object which is nullptr and *that* is causing a crash subsequently when the object is set. My suggestion would be (probably) to make it so that attempting to resolve a chunk in 'dest' mode (i.e. as a container) should throw an error if it doesn't exist.

Reply:
@runrevmark in the sample recipe `t_obj_chunk . object` is a card, so a simpler recipe to trigger a crash would be `put "foo" before card 1`. The updated patch throws an error in this case:

`Chunk: destination is not a container`, which matches the behavior of LC 6.x